test/y2storage/used_storage_features_test.rb

```diff
@@ -104,4 +104,16 @@
         .to contain_exactly("device-mapper", "multipath-tools", "fcoe-utils")
     end
   end
+
+  context "Created from integer representing UF_BTRFS" do
```

Comment:
I would not mention here specific flag, but to be real documention it is better to use something like `Created from integer representing used features`